src/components/ArticlePage.jsx

```diff
@@ -23,7 +27,21 @@ const ArticlePage = () => {
     });
   }, [article_id]);
 
-  console.log(comments[0]);
+  const handleCroak = (value) => {
+    if (value === "up") {
+      setLoadingUp(true);
+      patchArticleVotes(article_id, 1).then((res) => {
+        setVotes(votes + 1);
+        setLoadingUp(false);
```

Comment:
This isn't working optimistically - update the state first of setVotes then do the patch